docs/README.md

```diff
@@ -123,3 +130,420 @@ something like
+
+#### 2. First command
+
+We will now define our first [command](#commands), it will allow us to create posts in our
+blog. Write this command in the project's root directory:
+
+```bash
+boost new:command CreatePost --fields postId:UUID title:string content:string author:string
+```
+
+The `new:command` creates most of the code for us, take a look at it here:
+
+```text
+boosted-blog
+└── src
+    └── commands
+        └── CreatePost.ts
+```
+
+We still need to define two more things:
+
+1. Who is authorized to run this command.
+1. And what events are being triggered when the command is executed.
+
+We will create the first Event later on this guide so, let's start authorizing `'all'` to
+run this command, to do so edit `src/commands/CreatePost.ts` like this:
+
+```typescript
+@Command({
+  authorize: 'all', // Specify authorized roles here. Use 'all' to authorize anyone
+})
+export class CreatePost {
+  public constructor(
+    readonly postId: UUID,
+    readonly title: string,
+    readonly content: string,
+    readonly author: string
+  ) {}
+
+  public handle(register: Register): void {
+    register.events(/* YOUR EVENT HERE */)
+  }
+}
+
+```
+
+#### 3. First event
+
+It makes sense to emit an event after executing the command `CreatePost`, something like
+`PostCreated`. Let's do it by running this command:
+
+```bash
+boost new:event PostCreated --fields postId:UUID title:string content:string author:string
+```
+
+The `new:event` command generates a new file under the `src/events` directory. The name of
+the file is the name of the Event:
+
+```text
+boosted-blog
+└── src
+    └── events
+        └── CreatePost.ts
+```
+
+Edit the `entityID` method to return the id of the entity this Event is referring to. It
+should look like this:
+
+```typescript
+// src/events/PostCreated.ts
+
+@Event
+export class PostCreated {
+  public constructor(
+    readonly postId: UUID,
+    readonly title: string,
+    readonly content: string,
+    readonly author: string
+  ) {}
+
+  public entityID(): UUID {
+    return this.postId;
+  }
+}
+```
+
+Now that we have an event, we can make the `CreatePost` command to emit it. Let's change
+the command's `handle` method to look like this:
+
+```typescript
+// src/commands/CreatePost.ts::handle
+public handle(register: Register): void {
+  register.events(new PostCreated(this.postId, this.title, this.content, this.author))
+}
+```
+
+#### 4. First entity
+
+Now we have a command and an event, however, we do not have any data representation of a
+`Post`. To do so, we will create an `entity`.
+
+```bash
+boost new:entity Post --fields title:string content:string author:string --reduces PostCreated
+```
+
+This time Booster has created a file called `Post.ts` in the `src/entities` directory.
+An entity is a reduction function applied to a series of events. For this example, we won't
+take into account the previous state of the `Post`, so we will create a new one
+from the Event's data.
```

Comment:
Although it is not directly related with this PR, do we have in mind create other examples using the previous states?  I think that they will help a lot especially for more complex use cases.

Reply:
Thank you @rdiaz82 for pointing this out. Yep, we have already captured an examples section 👍